### ISSUE TITLE
zpool create example does not work

### ISSUE THREAD

**Issue by yurikoles** [openzfsonosx/zfs#398]
`./zpool.sh create BOOM /Users/lundman/openzfsonosx/diskimage.bin`
First of al, it's machine-specific. Second, there is missing step of file allocation,

**Comment by lundman** [comment]
Yes, this README has not been updated at all, the documentation has been placed in wiki, but this should be updated to reflect that
